enterprise/api/throttles.py

```diff
@@ -7,6 +7,7 @@
 from enterprise.api.utils import get_service_usernames
 
 SERVICE_USER_SCOPE = 'service_user'
+HIGH_SERVICE_USER_SCOPE = 'high_service_user'
 
 
 class ServiceUserThrottle(UserRateThrottle):
```

Comment:
We should probably use inheritance here to make this code DRY. We can introduce a parent class and inherit both of these classes from that class.